### PR TITLE
chore: patch-tsa-upgrade-test

### PR DESCRIPTION
**Pull request by JasonPowr**
Small fix the upgrade test, adds a retry to the GetTSACertificateChain() func

### CODE REVIEW

File: test/e2e/byodb_test.go [hunk 1]
```diff
@@ -164,7 +164,7 @@ var _ = Describe("Securesign install with byodb", Ordered, func() {
 
 			tsa := tas.GetTSA(ctx, cli, namespace.Name, securesign.Name)()
 			Expect(tsa).ToNot(BeNil())
-			err := tas.GetTSACertificateChain(ctx, cli, tsa.Namespace, tsa.Name, tsa.Status.Url)
+			err := tas.GetTSACertificateChain(ctx, cli, tsa.Namespace, tsa.Name, tsa.Status.Url, 5)
```
Comment (osmman) [inline test/e2e/byodb_test.go:167]: I think that it is better to use `Eventualy` function. Example: 
```go
Eventually(func() error {
  return tas.GetTSACertificateChain(ctx, cli, tsa.Namespace, tsa.Name, tsa.Status.Url)
}).To(Succeed())
```

File: test/e2e/support/tas/tsa.go [hunk 2]
```diff
@@ -53,15 +54,28 @@ func GetTSA(ctx context.Context, cli client.Client, ns string, name string) func
...
 }
 
-func GetTSACertificateChain(ctx context.Context, cli client.Client, ns string, name string, url string) error {
-	req, err := http.NewRequestWithContext(ctx, "GET", url+"/api/v1/timestamp/certchain", nil)
-	if err != nil {
-		return fmt.Errorf("failed to create HTTP request: %w", err)
+func GetTSACertificateChain(ctx context.Context, cli client.Client, ns string, name string, url string, retries int) error {
+	var resp *http.Response
+	var err error
+	for i := 0; i < retries; i++ {
```
Comment (osmman) [inline test/e2e/support/tas/tsa.go:60]: right now you can remove retries logic because it is handled by eventually

Comment (JasonPowr) [reply test/e2e/support/tas/tsa.go]: Yes ofc, thank you I was not thinking :)

**Pull request merged**